openshift/release/knative-eventing-kafka-broker-dp-ci.yaml

```diff
@@ -126,14 +126,14 @@ spec:
       name: kafka-broker-dispatcher
       labels:
         app: kafka-broker-dispatcher
-        kafka.eventing.knative.dev/release: devel
+        kafka.eventing.knative.dev/release: v1.1.0
     spec:
       serviceAccountName: knative-kafka-broker-data-plane
       securityContext:
         runAsNonRoot: false
       containers:
         - name: kafka-broker-dispatcher
-          image: registry.ci.openshift.org/openshift/knative-v1.1.3:knative-eventing-kafka-broker-dispatcher
+          image: registry.ci.openshift.org/openshift/knative-v1.1.0:knative-eventing-kafka-broker-dispatcher
```

Comment:
we are actually using 1.1.3 as the promotion:
https://github.com/openshift/release/blob/master/ci-operator/config/openshift-knative/eventing-kafka-broker/openshift-knative-eventing-kafka-broker-release-v1.1__49.yaml#L163

Reply:
Ok, thanks for helping me connect the dots.

would be better (easier to understand) make the promotion `knative-v1.1.0`?
For example, in other components, we don't care about the patch version.

Reply:
We usually do only care if there is upstream manifest changes in a `dot release`.

I can't exactly recall any specific reason, but I used the latest `1.1.x` when starting, and so nailed my WIP pr for S-O, to exactly that 1.1.3 version